operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -200,6 +200,16 @@ local utils = import 'mixin-utils/utils.libsonnet';
       ],
     },
 
+  // Creates a panel like queryPanel() but if the legend contains only 1 entry,
+  // than it configures the series alias color to the one used to display failures.
+  failurePanel(queries, legends, legendLink=null)::
+    $.queryPanel(queries, legends, legendLink) + {
+      // Set the failure color only if there's just 1 legend and it doesn't contain any placeholder.
```

Comment:
I understand that we don't configure color for legends with placeholders because we don't want them to be all same color.

Reply:
Correct.